apps/browser/src/tools/popup/send/send-groupings.component.ts

```diff
@@ -97,7 +96,7 @@ export class SendGroupingsComponent extends BaseSendComponent {
     }
 
     if (!this.syncService.syncInProgress || restoredScopeState) {
-      window.setTimeout(() => this.popupUtils.setContentScrollY(window, this.state?.scrollY), 0);
+      await BrowserPopupUtils.setContentScrollY(window, this.state?.scrollY);
```

Comment:
Adding await here changes the behavior of the method. With the plain setTimeout call, execution of the method continues. With the await, execution blocks until after the scroll state is set. The await should probably be removed here and in the other 3 places this method is called.

```suggestion
     BrowserPopupUtils.setContentScrollY(window, this.state?.scrollY);
```

Reply:
Updated